cloudinit/fix-dhcp.yml

```
@@ -7,4 +7,5 @@
 # TODO: Remove this code when and if that becomes possible.  See #96
 # for more details.
 runcmd:
-  - [netplan, apply]
+  - netplan
+  - apply
```

I believe this is what is necessary to get the same result as the original. As it is now I believe it will try to run `netplan` and then `apply` instead of `netplan apply`.

```suggestion
  - - netplan
    - apply
```

---

Good catch!